sk0-back/src/users/services/users.service.ts

```diff
@@ -55,8 +56,8 @@ export class UsersService {
     if (userByEmail && userByEmail.id !== id) {
       throw new UserExistException("Exists", 400);
     }
-
-    return this.usersRepository.save(user);
+    //TODO: use object mapper
```

Comment:
We can implement with class-transformer which is already installed

Reply:
Yesyes, we could/should. Let me check on that.